src/OpenCV/Features2d.hsc

```diff
@@ -285,6 +294,213 @@ orbDetectAndCompute orb img mbMask = unsafeWrapException $ do
+
+newSimpleBlobDetector :: SimpleBlobDetectorParams -> IO SimpleBlobDetector
+newSimpleBlobDetector SimpleBlobDetectorParams{..} = fromPtr
+    [CU.block|Ptr_SimpleBlobDetector * {
+      cv::SimpleBlobDetector::Params params;
+      params.blobColor           = $(unsigned char c'blobColor);
+      params.filterByArea        = $(bool c'filterByArea);
+      params.filterByCircularity = $(bool c'filterByCircularity);
+      params.filterByColor       = $(bool c'filterByColor);
+      params.filterByConvexity   = $(bool c'filterByConvexity);
+      params.filterByInertia     = $(bool c'filterByInertia);
+      params.maxArea             = $(float c'maxArea);
+      params.maxCircularity      = $(float c'maxCircularity);
+      params.maxConvexity        = $(float c'maxConvexity);
+      params.maxInertiaRatio     = $(float c'maxInertiaRatio);
+      params.maxThreshold        = $(float c'maxThreshold);
+      params.minArea             = $(float c'minArea);
+      params.minCircularity      = $(float c'minCircularity);
+      params.minConvexity        = $(float c'minConvexity);
+      params.minDistBetweenBlobs = $(float c'minDistBetweenBlobs);
+      params.minInertiaRatio     = $(float c'minInertiaRatio);
+      params.minRepeatability    = $(float c'minRepeatability);
+      params.minThreshold        = $(float c'minThreshold);
+      params.thresholdStep       = $(float c'thresholdStep);
+      cv::Ptr<cv::SimpleBlobDetector> detectorPtr =
+        cv::SimpleBlobDetector::create(params);
+      return new cv::Ptr<cv::SimpleBlobDetector>(detectorPtr);
+    }|]
+  where
+    c'filterByArea = fromBool blob_filterByArea
+    c'filterByCircularity = fromBool blob_filterByCircularity
+    c'filterByColor = fromBool blob_filterByColor
+    c'filterByConvexity = fromBool blob_filterByConvexity
+    c'filterByInertia = fromBool blob_filterByInertia
+    c'blobColor = fromIntegral blob_blobColor :: C.CUChar
+    c'maxArea             = realToFrac blob_maxArea
+    c'maxCircularity      = realToFrac blob_maxCircularity
+    c'maxConvexity        = realToFrac blob_maxConvexity
+    c'maxInertiaRatio     = realToFrac blob_maxInertiaRatio
+    c'maxThreshold        = realToFrac blob_maxThreshold
+    c'minArea             = realToFrac blob_minArea
+    c'minCircularity      = realToFrac blob_minCircularity
+    c'minConvexity        = realToFrac blob_minConvexity
+    c'minDistBetweenBlobs = realToFrac blob_minDistBetweenBlobs
+    c'minInertiaRatio     = realToFrac blob_minInertiaRatio
+    c'minRepeatability    = realToFrac blob_minRepeatability
+    c'minThreshold        = realToFrac blob_minThreshold
+    c'thresholdStep       = realToFrac blob_thresholdStep
+
+mkSimpleBlobDetector :: SimpleBlobDetectorParams -> SimpleBlobDetector
+mkSimpleBlobDetector = unsafePerformIO . newSimpleBlobDetector
+
+--------------------------------------------------------------------------------
+
+{- | Detect keypoints and compute descriptors
+
+Example:
+
+@
+@
```

Comment:
An example would be nice. But this may also be added later. 

Detecting coins is an example that often comes up. You could detect some coins as blobs and then draw the detected blobs as circles in the original image.